client/core/types.go

```diff
@@ -188,7 +273,7 @@ type Order struct {
 	Cancelling   bool              `json:"cancelling"`
 	Canceled     bool              `json:"canceled"`
 	FeesPaid     *FeeBreakdown     `json:"feesPaid"`
-	FundingCoins []dex.Bytes       `json:"fundingCoins"`
+	FundingCoins []*Coin           `json:"fundingCoins"`
```

Comment:
This and the the Match changes rule out 0.1.x unfortunately, which is no big deal as we'll move on to 0.2.0 right away.